### ISSUE TITLE
Sidebar and Sideboxes

### ISSUE THREAD

**Issue by IronRunes** [the-engine-room/alidade#24]
Introduce the left sidebar and sideboxed content. Implement CMS support for it.

**Comment by thomwithoutanh** [comment]
Left sidebar is currently displaying Step 4 and headings. 

Ideally, it would only display this step if the user triggered the conditional logic on step 3.6. Is this possible?

**Comment by IronRunes** [comment]
Yes, my bad for not explicitly pointing at issue #25 as a partial blocker for closing this.
